### DISCUSSION TITLE
SVG embedded PNG/JPG dimensions limit

### DISCUSSION THREAD

**Discussion by Melomate** [libvips/libvips#4013] [General]
When trying to convert a SVG which contains png/jpg images (for example resolution 15,000 X 15,000 px)
conversion result is just a blank image.

10,000 X 10,000px is working without issues

Can't tell whats the exact limit. Is it possible to remove that limit?

**Comment by lovell** [comment]
Are you using vips at the command line? Was it compiled with support for SVG input via librsvg? Do you see the same thing when using the `rsvg-convert` command line tool provided by librsvg?

If you still require help, please provide a sample image and an example of how you are processing it.

**Comment by Melomate** [comment]
Thanks for the fast reply.

`<?xml version="1.0" encoding="utf-8"?>
<svg height="1000" version="1.1" viewBox="0 0 1000 1000" width="1000" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<defs>
</defs>
<image height="100%" width="100%" x="0"  y="0" xlink:href="./id-prviifjpq.jpg"/>
</svg>`


This simple SVG for example.

I just tested it with rsvg-convert:

`rsvg-convert test.svg -o test.jpg`

I do indeed get a blank image too when using rsvg-convert.

Any idea how to proceed? Who maintains the librsvg repo?

**Reply by lovell** [reply]
> I do indeed get a blank image too when using rsvg-convert.

Please ensure you're using the latest version of librsvg as there have been many improvements recently.

If you're still having problems, you can ask upstream at https://gitlab.gnome.org/GNOME/librsvg/-/issues ensuring your question includes sample SVG and JPEG images that allow someone else to reproduce.
